Задание 3-2

ThewayofthesmuraiZero opened the pull request:
Задание  3-1
Протабулировать заданную в таблице функцию.
Использовать данные в таблице значения шага и интервала в качестве
ввода пользователя для решения тестового примера.
При невозможности расчёта функции в конкретной точке выводить
её значение и надпись, означающую отсутствие решения.
Вариант 11
![image](https://user-images.githubusercontent.com/70698521/113874080-16fa1280-97be-11eb-9e75-5b8ce1d789cb.png)



График в матлаб 
![image](https://user-images.githubusercontent.com/70698521/113874207-3a24c200-97be-11eb-84fd-bf91aea23991.png)
Полный график
![График](https://user-images.githubusercontent.com/70698521/113874352-64767f80-97be-11eb-9c1f-e3c9666fc961.png)



ЗАДАНИЕ 3-2
![image](https://user-images.githubusercontent.com/70698521/114271785-de577480-9a1b-11eb-8405-3cb8e8f52005.png)
ВАРИАНТ 11
![image](https://user-images.githubusercontent.com/70698521/114271797-eadbcd00-9a1b-11eb-8f33-c59fdd424427.png)

ThewayofthesmuraiZero committed:
Задание 3-1

Протабулировать заданную в таблице функцию.
Использовать данные в таблице значения шага и интервала в качестве
ввода пользователя для решения тестового примера.
При невозможности расчёта функции в конкретной точке выводить
её значение и надпись, означающую отсутствие решения.

MarinaPaley commented on Task 3-1.cpp:
```diff
@@ -0,0 +1,51 @@
+#include <limits>
+
+using namespace std;
+void  IsCalulted(const double x, double y);
```
Добавьте документацию


/**
 * \brief Вычисляет ширину прямоугольника
 * \param length Длина стороны прямоугольника.
 * \param factor Коэффициент пропорциональности между длиной
 * и шириной прямоугольника в %
 * \return Ширина прямоугольника
 */

MarinaPaley reviewed (approved):
Добавьте документацию
Запушьте исправления
После этого, меня можно не просить посмотреть, сразу делать `merge`

ThewayofthesmuraiZero committed:
Task 3-1

Внесла правки в код, добавила документацию

ThewayofthesmuraiZero committed:
Задание 3-2

Составьте две программы:
a)	вычислить сумму первых n членов последовательности
 (k = 1, 2, 3 ..., n).
b)	вычислить сумму всех членов последовательности, не меньших заданного
 числа e.
Помните о проверке пользовательского ввода. Все результаты вывести
на экран. Отчёт дополнить блок-схемой. При вычислении факториалов
рекомендуется отказаться от использования рекурсивных методов.

ВАРИАНТ 11

ThewayofthesmuraiZero committed:
Merge branch 'Feature/3' of https://github.com/ThewayofthesmuraiZero/C---programming into Feature/3

MarinaPaley commented on Task 3-2.cpp:
```diff
@@ -0,0 +1,86 @@
+
+double Sum(const size_t size)
+{
+    auto previous = 1.0 / (2 * 2 * 3);
```
= 8, нет? или 1, чтобы не считать

MarinaPaley commented on Task 3-2.cpp:
```diff
@@ -0,0 +1,86 @@
+
+double Recurrence(const size_t index)
+{
+    return (pow((index+1),3))/index ;
```
у меня другая формула получилась...

MarinaPaley reviewed (changes requested):
Для каждой задачи - свой `issue` и своя ветка!!!!
прошлую задачу нужно было вмержить, `PR` закрыть, ветку удалить

ThewayofthesmuraiZero committed:
Update Task 3-2.cpp

ThewayofthesmuraiZero committed:
Update Task 3-2.cpp
